provenance/utils.py

```diff
@@ -18,7 +18,7 @@ def args_extractor(f, merge_defaults=False):
     have a names.
 
     """
-    spec = getfullargspec(f)
+    spec = inspect.getfullargspec(f)
```

Comment:
So, by making this change I think this means `provenance` will only support python 3, is that right?